webui/react/src/components/Badge.module.scss

```diff
@@ -10,7 +10,12 @@
   text-align: center;
 }
 .id {
-  font-family: 'Source Code Pro', monospace;
+  font-family: "Source Code Pro", monospace;
```

Comment:
**nit:** I think most of our scss is using single quotes (fonts.scss, icons.scss). Should we add a rule to enforce one or the other?

Reply:
i added a rule for this it has auto format too